### PR TITLE
pikchr: init at unstable-2021-04-07

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by veprbl** [comment]
@GrahamcOfBorg eval

File: pkgs/tools/graphics/pikchr/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation {
+  pname = "pikchr";
+  version = "unstable-2021-04-07";
+
+  src = fetchurl {
+    url = "https://pikchr.org/home/tarball/90b6d5b4a3834ff0/pikchr.tar.gz";
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/pikchr/default.nix:11]: Does upstream over any easily versioned download?

File: pkgs/tools/graphics/pikchr/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    sha256 = "1cqpnljy12gl82rcbb7mwhgv9szcliwkbwwh67hzdv020h1scxym";
+  };
+
+  # can't open generated html files
+  patchPhase = ''
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/pikchr/default.nix:16]:
```suggestion
  postPatch = ''
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/graphics/pikchr/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/tools/graphics/pikchr/default.nix:11]: As far as I understand, tags should appear [here](https://pikchr.org/home/taglist), but there isn't any, so no.

File: pkgs/tools/graphics/pikchr/default.nix (hunk 2 above)
Comment (fgaz) [reply pkgs/tools/graphics/pikchr/default.nix]: Fixed

**Comment by fgaz** [comment]
Ping

**Pull request merged**